ext/opcache/jit/ir/ir_aarch64.dasc

```diff
@@ -5337,9 +5337,6 @@ void *ir_emit_code(ir_ctx *ctx, size_t *size_ptr)
 	if (ret != DASM_S_OK) {
 		IR_ASSERT(0);
 		dasm_free(&data.dasm_state);
-		if (ctx->code_buffer == NULL) {
-			ir_mem_unmap(entry, size);
-		}
```

Comment:
This ``unmap`` should never be executed in PHP, because PHP should always set ``ctx->code_buffer``.
May we have a different bug, but the existing code looks right.

Reply:
Thanks for letting me know. I think, at lease, we should keep this code consistent with `x86.dasc`.

Reply:
Oh, your patch is right of course. Sorry!
I'll take care to commiting it into IR and than merge to PHP.